node/src/runtime/mainnet.rs

```diff
@@ -85,6 +85,8 @@ pub fn testnet_genesis(
 		staking: StakingConfig {
 			validator_count: 100,
 			minimum_validator_count: 1,
+			min_validator_bond: 1000 * GGX,
```

Comment:
@akorchyn While this file is named "mainnet", is it still corresposnds to Sydney?

Reply:
Mainnet is a Sydney from runtime perspective.

We have two runtimes, one with staking and POS. It's named mainnet. Another one is POA without staking, tokenomics. It's called testnet.

I know it's messy but currently prod means Sydney across all code base. And testnet means Brooklyn.